app/mailers/candidate_mailer.rb

```diff
@@ -73,6 +73,10 @@ def application_rejected_awaiting_decisions(application_choice)
     application_rejected(application_choice, :awaiting_decisions)
   end
 
+  def application_rejected_offers_made(application_choice)
```

Comment:
I'm wondering if these methods should be a little clearer, it took me a short while to work out what was going on here.
Maybe: `application_rejected_but_with_other_offers_made`

Reply:
We're going to do a follow up PR to refactor the previous PRs, we'll take this onboard.